frontend/src/GameReports.tsx

```diff
@@ -49,6 +50,8 @@ const TABLE_TOP_POSITION =
     TABLE_BTN_HEIGHT_PX +
     TABLE_ELEMENTS_GAP * 2;
 
+const PAGE_LIMIT = 100;
```

Comment:
Want to make this a dropdown menu in the future - will make a low priority issue for it